nemo/collections/tts/parts/utils/tts_dataset_utils.py

```diff
@@ -61,7 +66,7 @@ def get_audio_filepaths(manifest_entry: Dict[str, Any], audio_dir: Path) -> Tupl
     return audio_filepath_abs, audio_filepath_rel
 
 
-def normalize_volume(audio: np.array, volume_level: float) -> np.array:
+def normalize_volume(audio: np.array, volume_level: float = 0.95) -> np.array:
```

Comment:
Just one little thing, feel free to skip

Digital volume is commonly measured is dBFS. 0.95 max absolute sample value corresponds to 20*log10(0.95)=-0.44 dBFS.

Would be nice to see this acknowledged in the docstring